js/script.js

```diff
@@ -1169,19 +1169,19 @@ var validCombos = {
         "Follow build instructions from <a href='https://aka.ms/build-ort-snpe' target='_blank'>here</a>",
 
     //Xnnpack
-    "android,C-API,ARM64,XNNPACK":
+    "ios,android,C-API,ARM64,XNNPACK":
```

Comment:
This is an invalid format. iOS entry will need a new entry. Please stage this for preview to test: https://github.com/microsoft/onnxruntime/wiki/Update-ONNX-Runtime-Docs#preview-change

Reply:
Thanks.
Provided the [link](https://wejoncy.github.io/onnxruntime/) in description part.